lib/ansible/modules/system/aix_inittab.py

```diff
@@ -0,0 +1,239 @@
+requirements: [ 'itertools']
+'''
+
+EXAMPLES = '''
+# Add service startmyservice to the inittab, directly after service existingservice.
+- name: Add startmyservice to inittab
+  aix_inittab:
+    name: startmyservice
+    runlevel: 4
+    action: once
+    command: "echo hello"
+    insertafter: existingservice
+    state: present
+  become: yes
+
+# Change inittab enrty startmyservice to runlevel "2" and processaction "wait".
+- name: Change startmyservice to inittab
+  aix_inittab:
+    name: startmyservice
+    runlevel: 2
+    action: wait
+    command: "echo hello"
+    state: present
+  become: yes
+
+# Remove inittab entry startmyservice.
+- name: remove startmyservice from inittab
+  aix_inittab:
+    name: startmyservice
+    runlevel: 2
+    action: wait
+    command: "echo hello"
+    state: absent
+  become: yes
+'''
+
+RETURN = '''
+# description: The result is deliverd in an dictionary.
+- return:
+  changed: true
+  msg: []
+  name: "startmyservice"
+  status: "changed inittab entry startmyservice"
+  warnings: []
+'''
+
+# Import necessary libraries
+import itertools
+from ansible.module_utils.basic import AnsibleModule
+
+# end import modules
+# start defining the functions
+
+
+def check_current_entry(module):
+    # Check if entry exists, if not return False in exists in return dict,
+    # if true return True and the entry in return dict
+    existsdict = { 'exist' : False }
+    (rc, out, err) = module.run_command(['lsitab', module.params['name']])
+    if rc == 0:
+        keys = ('name', 'runlevel', 'action', 'command')
+        values = out.split(":")
+        values = map(lambda s: s.strip(), values)  # strip non readable characters as \n
+        existsdict = dict(itertools.izip(keys,values))
+        existsdict.update({ 'exist' : True })
+    return existsdict
+
+
+def main():
+    # initialize
+    module = AnsibleModule(
+        argument_spec = dict(
+            name = dict( required=True, type='str', aliases=['service']),
+            runlevel = dict( required=True, type='str'),
+            action = dict( choices=[
+                           'respawn',
+                           'wait',
+                           'once',
+                           'boot',
+                           'bootwait',
+                           'powerfail',
+                           'powerwait',
+                           'off',
+                           'hold',
+                           'ondemand',
+                           'initdefault',
+                           'sysinit'
+                           ], type='str'),
+            command = dict( required=True, type='str' ),
+            insertafter = dict( type='str' ),
+            state = dict( choices=[
+                          'present',
+                          'absent',
+                          ], required=True, type='str'),
+        )
+        )
+
+    result = {
+        'name':  module.params['name'],
+        'changed': False,
+        'status': {},
```

Comment:
It seems that `status` is used as a normal string, but it was defined here as a dictionary ?

Also, you don't have to initialize these entries here, you can simply assign them at will. Unless you have cases where you prefer to return an empty string rather than no entry.